pennylane/gradients/parameter_shift.py

```diff
@@ -438,6 +454,33 @@ def _reorder_grad_axes_multi_measure(
     return new_grad
 
 
+def _reorder_grads(grads, tape_specs):

```

Comment:
Is it useful in any other gradient transforms?

Reply:
Yep, it's used in the `stoch_pulse_grad`.

Reply:
Should I make it public?

Reply:
Up to you, I think it is a good occasion to move it and promote it :+1: